core/src/CoreTelemetryService.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.Identity.Core
+{
+    internal class CoreTelemetryService
+    {
+        public static void InitializeCoreTelemetryService(ITelemetry instance)
+        {
+            Instance = instance;
```

Comment:
>instance [](start = 23, length = 8)

pls. ensure this is initialized in the ModuleInitalizer

Reply:
#Resolved